config/musonza_chat.php

```diff
@@ -2,6 +2,7 @@
 
 return [
     'user_model' => 'App\User',
+    'db_connection' => 'mysql',
```

Comment:
or maybe
```suggestion
    'db_connection' => env('CHAT_DB_CONNECTION', 'mysql'),
```

Reply:
OK, set db_connection to null.
If not set, change to use default